sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -290,6 +291,44 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       checkAnswer(
         sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
         sql("SELECT * FROM non_part_table").collect())
+
+      // check for case insensitive property of partition column name in load command.
+      withSQLConf(SQLConf.CASE_SENSITIVE.key -> "false") {
+        withInputFile { f =>
+          sql(s"""$loadQuery INPATH "${f.toURI}" INTO TABLE part_table PARTITION(C="1", D="2")""")
+        }
+        checkAnswer(
+          sql("SELECT employeeID, employeeName FROM part_table WHERE c = '2' AND d = '1'"),
+          sql("SELECT * FROM non_part_table").collect())
```

Comment:
nit: you don't need `collect()`.

Reply:
Is this LOAD issue specific to Hive? If not, could you move this test to `DDLParserSuite.scala`?

Reply:
nit: you don't need collect(). Fixed

Reply:
> Is this LOAD issue specific to Hive? If not, could you move this test to `DDLParserSuite.scala`?

Yes, i think Load commands are specific to Hive tables.

Reply:
oh, really? https://github.com/apache/spark/blob/a3bbc371cbdbcb469a181de8f58f83e193d4f341/sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala#L1505

Reply:
you want me to move this testcase to DDLParserSuite.scala?

Reply:
> oh, really?
> 
> https://github.com/apache/spark/blob/a3bbc371cbdbcb469a181de8f58f83e193d4f341/sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala#L1505

@maropu  sorry for misinterpettation, i moved the testcase to DDLParserSuite.scala, please let me know for any further clarifications. thanks for all valuable inputs.

Reply:
@maropu @dongjoon-hyun 
There is a small problem, DDLParserSuite is a suite which basically does the verification of  parsing and plan resolution, no where i can find a testcase with execution flow, In my case since the resolution happens while Load command execution,  so i am not sure we shall use this suite for verifiction of this usecase, please suggest . Thanks